low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -22,28 +22,233 @@
  ***************************************************************************
 """
 
+import errno
+import time
+import os
+import re
+import psutil
+import socket
 from resource_map import ResourceMap
+from error import ResourceMapError
+
+from framework.utils.tool_factory import ToolFactory
 
 
 class ServerMap(ResourceMap):
     """Provides server resource related information."""
 
     name = "server"
+    SAS_RESOURCE_ID = ["SASHBA-0"]
+    NW_CBL_CARRIER_FILE = "/sys/class/net/%s/carrier"
+    NW_CBL_OPERSTATE_FILE = "/sys/class/net/%s/operstate"
 
     def __init__(self):
         """Initialize server"""
-        pass
+        super().__init__()
+        self.server_frus = {
```

Comment:
should we have a separation of hw and software here @mariyappanp ?? I tried to keep it aligned with storage.py but this was the though I had